Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -29,6 +29,26 @@ def test_load_module_API(self):
         with self.assertRaises(ImportError):
             self.load_module('XXX')
 
+    def test_deprecation_warning_argument_name_and_fullname_use(self):
+        with self.assertWarns(DeprecationWarning) as cm:
+            self.machinery.ExtensionFileLoader(
+                    name=util.EXTENSIONS.name,
+                    path=util.EXTENSIONS.file_path)
+
+        self.assertIn("the 'name' parameter is deprecated; use 'fullname' "
+                      "instead", str(cm.warning))
+
+    def test_argument_name_and_fullname_use(self):
+        with self.assertWarns(DeprecationWarning) as warning_cm, \
+              self.assertRaises(TypeError) as exception_cm:
+            self.machinery.ExtensionFileLoader(
+                fullname=util.EXTENSIONS.name,
+                name=util.EXTENSIONS.name,
+                path=util.EXTENSIONS.file_path)
+
+        self.assertIn("'name' and 'fullname' cannot be used simultaneously",
+                      str(exception_cm.exception))
```

Comment:
```suggestion
        self.assertIn("fullname", str(exception_cm.exception))
```
You typically don't want to assert exception messages as it means you can't change them easily. Since they are not generally considered part of the API the most you typically want to do is make sure any critical detail happens to be mentioned and then leave the rest alone. Same applies above in the other test.

Reply:
@brettcannon makes sense, I updated it as @berkerpeksag suggested it would be a good choice to test the message. I'll revert it back.